website/docs/feature-flag-tutorials/use-cases/a-b-testing.md

```diff
@@ -222,13 +223,10 @@ The variants you created with Unleash would be the "arms" in the multi-bandit co
 
 To use Unleash to conduct a multi-arm bandit test, follow these steps:
 
-1. Collect the necessary data from each variant’s performance by enabling impression data for your feature flag
-2. Capture impression events in your application code
-3. Funnel the impression events captured from your application code to an external analytics tool
-4. Create [signal endpoints](/reference/signals) in Unleash and point them to your external analytics tools
-5. Create [actions](/reference/actions) in Unleash that can react to your signals
-
-Learn how to configure [actions](/reference/actions) and [signals](/reference/signals) from our documentation to get started.
+1. Collect the necessary data from each variant’s performance by enabling impression data for your feature flag.
+2. Capture impression events in your application code.
+3. Funnel the impression events captured from your application code to an external analytics tool.
+4. Use the Unleash API to dynamically adjust the traffic for each variant based on performance.
```

Comment:
@nnennandukwe @sebastian-bury I kept it high level, do you think it works?

Reply:
Yeah I like this. Keeping it general gives them room to decide how they want to go out about the details specific to their needs, while pointing them in the right direction to at least get set up / started with the features Unleash has for it.